data/backend/authentication/models.py

```diff
@@ -0,0 +1,12 @@
+import uuid
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+from mailauth.contrib.user.models import AbstractEmailUser
+
+
+class EcobalyseUser(AbstractEmailUser):
+    organization = models.CharField(_("Company"), max_length=150, blank=True, null=True)
+    terms_of_use = models.BooleanField(default=False)
+    token = models.CharField(
+        _("TOKEN"), max_length=36, default=uuid.uuid4, editable=False
+    )
```

Comment:
I'm confused as `mailauth.contrib.user` is disabled from `INSTALLED_APPS` in the settings… here we import a model from a disabled app, that can't work I think.

May explain the `sqlite3.OperationalError: no such table: authentication_ecobalyseuser` error I get whenever trying to interact with the authentication views?

Reply:
It's just a way to redefine a model in another app. The other app is installed, not the original. The OperationalError is due to a missing makemigrations && migration